### ISSUE TITLE
Koebner Phenomenon

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10577]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 31971748 The Koebner phenomenon (KP), first described in 1876 by Heinrich Koebner, is the appearance of new skin lesions on previously unaffected skin secondary to trauma. This phenomenon is also termed the isomorphic (from Greek, “equal shape”) response, given the fact that the new lesions that appear are clinically and histologically identical to the patient’s underlying cutaneous disease.(....) 
The Koebner phenomenon is not to be confused with pathergy, which is a term utilized to describe when minor trauma (including blunt trauma such as bumps and bruises) can induce skin lesions.

PMID: 30855928 The rash in AOSD is classically characterized as evanescent, displaying a salmon-colored, macular, or maculopapular appearance. It is usually nonpruritic and tends to coincide with the onset of fever. While primarily observed on the trunk and extremities, the rash can manifest on the palms, soles, and face. The rash can sometimes be induced by heat (hot shower or towel) or skin friction, known as the Koebner phenomenon.

**Parent term (use hpo.jax.org/app)**
Abnormality of skin physiology HP:0011122

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:829  Adult-onset Still disease
Pellagra ORPHA:97352
**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000933
- label: Koebner Phenomenon
- parents: [Abnormality of skin physiology (HP:0011122)](https://hpo.jax.org/app/browse/term/HP:0011122)
- pmid: PMID:31971748
- def: The appearance of new skin lesions on previously unaffected skin secondary to trauma. This phenomenon is also termed the isomorphic (from Greek, equal shape) response, given the fact that the new lesions that appear are clinically and histologically identical to the patient's underlying cutaneous disease.
- comment: The Koebner phenomenon is typically associated with preexisting skin disease. Clinical entities that commonly display the Koebner phenomenon include psoriasis, vitiligo, lichen planus. The incidence of Koebnerization ranges broadly within these entities; estimates of the prevalence of the Koebner phenomenon are approximately 11 to 75% in psoriasis and 21 to 62% in vitiligo.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10577
- synonyms:
